frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -76,7 +78,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
     id: state.id,
     csrfToken,
     meta,
-    defaultState: state.personalInformation,
+    defaultState: { ...state.personalInformation, email: state.communicationPreferences?.email ?? state.personalInformation?.email, confirmEmail: state.communicationPreferences?.confirmEmail ?? state.personalInformation?.confirmEmail },
```

Comment:
Reverse the order. Only use `communicationPreferences?.email` if `personalInformation?.email` is not defined. Otherwise the user might enter an email here but a different email in the other page then come back to personal information page and his email will be overriden by the communication page. It's confusing.